pkg/rendering/renderer.go

```diff
@@ -344,6 +346,10 @@ func injectValuesOverrides(values *Values, mch *v1.MultiClusterHub, images map[s
 
 	values.HubConfig.OCPIngress = os.Getenv("INGRESS_DOMAIN")
```

Comment:
OCPIngress would be necessary at global level, if we used `baseDomain` it would require no changes on our side but global.OCPIngress is also ok.

Reply:
This is addressed now, thanks.!